site/public/js/ta-grading-v2.js

```diff
@@ -932,3 +932,51 @@ function findAllOpenedFiles(elem, current_path, path, stored_paths, first) {
 
   return stored_paths;
 }
+
+function getNonAnonPath(path, anon_submitter_id, user_ids){
+  let nonAnonPath = "";
+  let pathPieces = path.split("/");
+  for (i = 1; i < pathPieces.length; i++) {
+    if(i == 9){
```

Comment:
Why is `i=9` a special case can you leave a comment here or define a constant so its more clear whats going

Reply:
It's actually the main part of this function as its converting the anonymized path back to non-anonymized path... added :)